website/www/site/layouts/shortcodes/documentation/sdks.html

```diff
@@ -0,0 +1,18 @@
+{{/*
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. See accompanying LICENSE file.
+   */}}
+
+   {{ $data := index $.Site.Data .Site.Language.Lang }}
+   <div class="documentation-list">
+    {{ range $item := $data.documentation_sdks }}
+        {{ partial "documentation/icon-row" (dict "logo" .icon  "header" .name "text" .description) }}
+   {{ end }}
```

Comment:
The icons aren't showing up for the SDKs for me: 
![image](https://user-images.githubusercontent.com/675055/106774165-b2240f80-65f6-11eb-8186-08443e197490.png)

I can't figure out why the icons are working for the runners and not the SDKs, do you know?

Reply:
Thanks this is fixed now.